lib/monitoring/middleware/conjur_exporter.rb

```diff
@@ -0,0 +1,94 @@
+require 'prometheus/client'
+require 'prometheus/client/formats/text'
+
+module Monitoring
+  module Middleware
+    # Exporter is a Rack middleware that provides a sample implementation of a
+    # Prometheus HTTP exposition endpoint.
+    #
+    # By default it will export the state of the global registry and expose it
+    # under `/metrics`. Use the `:registry` and `:path` options to change the
+    # defaults.
+    class ConjurExporter
```

Comment:
Thinking we should rename this `PrometheusExporter` to more accurately capture what it represents